source/plugin/live.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Live
+group: Plugins
+order: 14
+---
+
+Use live plugin to animate cells and rows when they will be changed.
+
+
+{% docEditor "github/qgrid/ng2-example/tree/live-data-basic/latest" %}
+
+## Installation
+
+- For cell animation: Add `ng-template` inside `q-grid-column` with `change` input and add live-cell component inside your `ng-template`.
+
+- For row animation: Add `q-grid-live-rows` inside of q-grid component.
+
+```typescript
+@Component({
+   selector: 'my-component',
+   template: `
+        <q-grid [rows]="rows">
+	        <q-grid-columns generation="deep">
+		        <q-grid-column key="last" type="currency">
+			        <ng-template for="change" let-$cell>
+				        <q-grid-live-cell
+					        [cell]="$cell">
+				        </q-grid-live-cell>
+		        	</ng-template>
+		        </q-grid-column>
+            </q-grid-columns>
+			<q-grid-live-rows></q-grid-live-rows>
+         </q-grid>
+    `
+})
+export class MyComponent {
+	rows: Quote[];
```

Comment:
Use ViewChild like in other documentation, this piece should not be matched with example exactly